src/sqlfluff/core/rules/base.py

```diff
@@ -734,6 +795,7 @@ def valid_rule_name_regex(self):
         The second group captures the rule code.
 
         Examples of valid rule names:
+
```

Comment:
Unrelated fix: Need a blank line here in order for Sphinx to render the bullet points correctly.